e2e/core/settings-component.e2e.ts

```diff
@@ -0,0 +1,248 @@
+            expect(settingsPage.getBpmHostUrl()).toEqual(TestConfig.adf.url, 'The BPM Settings changes are not saved');
+            expect(settingsPage.getEcmHostUrl()).toEqual(TestConfig.adf.url, 'The ECM Settings changes are not saved');
+
+        });
+
+        it('[C291949] Should have field validation for Content Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmAndBpmOption(), 'ALL');
+            settingsPage.clearContentServicesURL();
+            settingsPage.ecmText.sendKeys(protractor.Key.TAB);
+            settingsPage.checkValidationMessageIsDisplayed();
+            settingsPage.checkApplyButtonIsDisabled();
+        });
+
+        it('[C291950] Should have field validation for Process Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmAndBpmOption(), 'ALL');
+            settingsPage.clearProcessServicesURL();
+            settingsPage.bpmText.sendKeys(protractor.Key.TAB);
+            settingsPage.checkValidationMessageIsDisplayed();
+            settingsPage.checkApplyButtonIsDisabled();
+        });
+
+        it('[C291951] Should not be able to sign in with invalid Content Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmOption(), 'ECM');
+            settingsPage.setContentServicesURL('http://localhost:7070');
+            settingsPage.clickApply();
+            loginPage.waitForElements();
+            loginPage.enterUsername(adminUserModel.id);
+            loginPage.enterPassword(adminUserModel.password);
+            loginPage.clickSignInButton();
+            expect(loginPage.getLoginError()).toMatch(loginError);
+        });
+
+        it('[C291952] Should not be able to sign in with invalid Process Services Url', () => {
+            settingsPage.setProvider(settingsPage.getBpmOption(), 'BPM');
+            settingsPage.setProcessServicesURL('http://localhost:7070');
+            settingsPage.clickApply();
+            loginPage.waitForElements();
+            loginPage.enterUsername(adminUserModel.id);
+            loginPage.enterPassword(adminUserModel.password);
+            loginPage.clickSignInButton();
+            expect(loginPage.getLoginError()).toMatch(loginError);
+        });
+    });
+
+    describe('Settings Component - BPM Basic Authentication', () => {
+        beforeAll( (done) => {
+            settingsPage.goToSettingsPage();
+            settingsPage.setProvider(settingsPage.getEcmAndBpmOption(), 'ALL');
+            settingsPage.setContentServicesURL(TestConfig.adf.url);
+            settingsPage.setProcessServicesURL(TestConfig.adf.url);
+            settingsPage.clickApply();
+            done();
+        });
+
+        it('[C277751] Should allow the User to login to Process Services using the BPM selection on Settings page', () => {
+            loginPage.goToLoginPage();
+            loginPage.clickSettingsIcon();
```

Comment:
These two steps are at the beginning of every test, I think it would be better to put them in beforeEach().

Reply:
done